mriqc/interfaces/common.py

```diff
@@ -257,3 +257,45 @@ def _run_interface(self, runtime):
             self._results['out_mask'] = out_mask
 
         return runtime
+
+
+class CopyImageHeaderInformationInputSpec(ANTSCommandInputSpec):
+
+    reference_image = File(exists=True, mandatory=True, position=1, argstr='%s',
+                           desc='reference image file')
+    input_image = File(exists=True, mandatory=True, position=2, argstr='%s',
+                       desc='input image file to copy ref image info to')
+    output_image = File(exists=False, mandatory=False, position=3, argstr='%s',
+                        name_source=['input_image'], name_template='%s_updated', desc='output image file',
+                        keep_extension=True)
+    copy_direction = traits.Bool(False, usedefault=True, position=4, argstr='%d',
+                                      desc='boolean, copy direction if True')
+    copy_origin = traits.Bool(False, usedefault=True, position=5, argstr='%d',
+                                   desc='boolean, copy origin if True')
+    copy_spacing = traits.Bool(False, usedefault=True, position=6, argstr='%d',
+                                    desc='boolean, copy spacing if True')
+
+
+class CopyImageHeaderInformationOutputSpec(TraitedSpec):
+    output_image = traits.File(exists=True, desc='updated file')
+
+
+class CopyImageHeaderInformation(ANTSCommand):
```

Comment:
Where is this used for? - If the intent is to revert the nasty habit of ANTs of changing the headers, then please consider whether any of these options - https://github.com/poldracklab/niworkflows/blob/master/niworkflows/interfaces/fixes.py will suffice

Reply:
Could you submit this interface to niworkflows?

Reply:
looks like you did :)